docusaurus/docs/iOS/guides/working-with-attachments.md

```diff
@@ -1,3 +1,263 @@
 ---
 title: Working with Attachments/Files
----
\ No newline at end of file
+---
```

Comment:
As far as I understood from this PR: https://github.com/GetStream/stream-chat-swift/pull/1198 We now need also Id's for each document right? @DominikBucher12

Reply:
Ah okay, nevermind then 👍